flit_core/flit_core/common.py

```diff
@@ -416,3 +416,20 @@ def normalize_dist_name(name: str, version: str) -> str:
 def dist_info_name(distribution, version):
     """Get the correct name of the .dist-info folder"""
     return normalize_dist_name(distribution, version) + '.dist-info'
+
+
+def walk_data_dir(data_directory):
+    """Iterate over the files in the given data directory.
+
+    Yields absolute paths - caller may want to make them relative.
```

Comment:
If the argument passed in is `./foo`, will it expand that to `/Users/pradyunsg/Developer/magic-project/foo`? If not, I think we might want to rephrase this to say paths are always prefixed with `data_directory` or soemthing like that.

Reply:
Yup, good point.